cmd/capibmadm/utils/utils.go

```diff
@@ -26,14 +26,40 @@ import (
 	"github.com/IBM/go-sdk-core/v5/core"
 	"github.com/IBM/platform-services-go-sdk/iamidentityv1"
 	"github.com/IBM/platform-services-go-sdk/resourcemanagerv2"
+	"github.com/IBM/vpc-go-sdk/vpcv1"
 
-	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/platformservices"
 	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/options"
 )
 
-// GetAccountID returns IBM cloud account ID of API key used.
+// CreateVPCService Function to create vpc service.
+func CreateVPCService(region string) (*vpcv1.VpcV1, error) {
```

Comment:
This can be removed since, [client pkg](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/cmd/capibmadm/clients) has all the necessary funcs.